src/main/resources/assets/admin/common/styles/api/form/set/form-set-view.less

```diff
@@ -1,207 +1,216 @@
     }
 
-    > .bottom-button-row {
-      display: none;
+    & > .validation-viewer {
+      background-color: @admin-bg-light-gray;
+      padding: 0 10px 10px 10px;
+      margin-left: 0 !important;
     }
 
-  }
+    &.max-1-occurrence {
+      .occurrence-views-container {
+        > .form-option-set-occurrence-view {
+          margin-bottom: 0;
+        }
 
-  .occurrence-views-container {
+        &:empty + .bottom-button-row {
+          display: block;
+          padding-left: 0;
+          padding-bottom: 0;
+        }
+      }
 
-    &:empty {
-      display: none;
-    }
+      > .bottom-button-row {
+        display: none;
+      }
 
-    .form-option-set-occurrence-view {
-      padding: 10px;
     }
 
-    .form-item-set-occurrence-view, .form-option-set-occurrence-view {
-      clear: left;
-      margin: 0;
-      margin-bottom: 10px;
-      background-color: white;
+    .occurrence-views-container {
 
-      &.collapsed {
-        .form-occurrence-draggable-label {
-          cursor: zoom-in;
-        }
+      &:empty {
+        display: none;
       }
 
-      .form-occurrence-draggable-label {
-        font-size: 14px;
-        color: #777777;
-        cursor: zoom-out;
-
-        .required {
-          &:after {
-            content: '*';
-            color: @input-red-text;
+      .form-option-set-occurrence-view {
+        padding: 10px;
+      }
+
+      .form-item-set-occurrence-view, .form-option-set-occurrence-view {
+        clear: left;
+        margin: 0;
+        border: 2px solid @admin-font-gray;
+
+        &.collapsed {
+          .form-occurrence-draggable-label {
+            cursor: zoom-in;
           }
         }
 
-        .drag-control {
-          height: 20px;
-          width: 10px;
-          background-position-x: center;
-          padding-right: 0;
-          margin-right: 8px;
+        & > .remove-button {
+          &:extend(.icon);
+          .icon-close();
+          .focusable();
+
+          margin-top: 3px;
+          margin-right: -4px;
+          border: 1px solid transparent;
+          line-height: 12px;
+          float: right;
+          text-decoration: none;
+          cursor: pointer;
         }
-      }
 
-      & > .remove-button {
-        &:extend(.icon);
-        .icon-close();
-        .focusable();
+        .form-occurrence-draggable-label {
+          display: inline-block;
+          padding-right: 0;
+          font-size: 14px;
+          color: @admin-font-gray2;
+          cursor: zoom-out;
 
-        margin: 5px 0 0 0;
-        border: 1px solid transparent;
-        line-height: 12px;
-        float: right;
-        text-decoration: none;
-        cursor: pointer;
-      }
+          .required {
+            padding-left: 5px;
 
-      .form-option-set-occurrences-container {
-        padding-bottom: 10px;
-      }
+            &:after {
+              content: '*';
+              color: @input-red-text;
+            }
+          }
+
+          .drag-control {
```

Comment:
Codacy found an issue: [Expected selector ".form-view .form-item-set-view .occurrence-views-container .form-item-set-occurrence-view .form-occurrence-draggable-label .drag-control" to come before selector ".form-view .form-item-set-view.multiple-occurrence > .occurrence-views-container > .form-item-set-occurrence-view > .form-occurrence-draggable-label > .drag-control" (no-descending-specificity)](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6371096)